### DISCUSSION TITLE
Will the PL18EV be every supported by EdgeTX?

### DISCUSSION THREAD

**Discussion by crasy666** [EdgeTX/edgetx#2003] [General]
Hi,

I noticed, that the team is testing EdgeTX on the[FlySky PL18 & Paladin EV. Do you see a chance that this will work?

the hardware of teh PL18EV is from my point of view great for excavators and co, but the Software is unusable, no complex curves or other pump mixers....

I am using actually 4 TX16S all with EdgeTX, which is ok, but not perfect for a excavator, I miss the two sliders for the drive control...

https://github.com/EdgeTX/edgetx/projects/11

Regards
Wolfgang

**Comment by raphaelcoeffic** [comment]
We do have some things working. That being said, there is still some way to have it really supported. And to get the feature set, some HW mods will be required, especially for:
- storage (no SD card interface)
- audio (only pre-installed samples)

**Comment by pfeerick** [comment]
Just further on the support for the PL18 (and PL18EV)... here is the guide on how to get started with a test version of EdgeTX ... https://github.com/EdgeTX/edgetx/wiki/Flashing-EdgeTX-to-Flysky-PL18-or-Paladin-EV

**Comment by brainbubblersbest** [comment]
IO have another Thing in Mind:
RC Cars!
Imagine the first EDGETX RC Car Radio supported by Flysky and EdgeTX.
Is anything relatable Part of a Conversation?
[Noble Pro as an Example](https://www.flysky-cn.com/noble-pro-description-2)

**Reply by pfeerick** [reply]
At the moment the focus in their more air and general purpose radios, but I also hope for the NB4 in the future. Another manufacturer is looking at a car/pistol grip radio with ETX, so it should happen one way or another ;)

**Comment by Poppy-Ann** [comment]
Hi there,  I came across this page whilst searching for a way to fix my pl 18ev which has thrown a wobbler and froze up whilst programing a model in it all it will do is turn on and off and show one page where I can see the 3 channels I programmed in and when I move the stick for the channels programmed it shows the position of the servos but no matter what I press or swipe on the screen it does not respond I have tried everything I can think of but I am stuck and when I came across this page I thought that if anyone can help it will be someone who knows how to program it, any help would be appreciated, if this is not where I should be requesting help then I am sorry.  Regards Poppy Ann

**Comment by rotorman** [comment]
EdgeTX nightly firmware has of now limited support for Flysky PL18 and Paladin EV radios. Flashing is possible via EdgeTX Buddy.
Limited support due to the fact that Flysky PL18 and Paladin EV radio hardware differs somewhat from other EdgeTX supported radios. Read https://github.com/EdgeTX/edgetx/wiki/PL18-%26-PL18EV-Hardware-Mod-for-Complete-EdgeTX-Support to understand the hardware limitations of EdgeTX on PL18/PL18EV and to read about possible optional hardware modifications that you can carry out to remedy some of them (voiding warranty!).
We would appreciate Flysky PL18 and Flysky Paladin EV (=PL18EV) radio users to test EdgeTX firmware and flashing. Please be aware that this is not a release or not even a release candidate status firmware yet, thus do limit the usage to avoid possible damage/harm. E.g. do not go fly a model yet with this firmware. Also please be aware that Flysky is using a variety of different LCD display types on PL18 and Paladin EV radios and our current code might not support correctly all of them! In case you might happen to have a PL18 or PL18EV with one of those yet unsupported displays and in order for you not to be left with a "bricked" system, be very sure to backup the original bootloader as written in https://github.com/EdgeTX/edgetx/wiki/Flashing-EdgeTX-to-Flysky-PL18-or-Paladin-EV , as only then you are able to at least return back to original Flysky firmware.
Please also be aware that you will lose all your settings and models that were done under original Flysky system - there is no conversion from Flysky system to EdgeTX system. You will start with a clean state.

**Reply by DantesInferi** [reply]
What's with the today announcement from official FlySkyRC ?- 
"Hi Flysky fans! We're excited to announce that Paladin PL18 and Paladin EV are compatible with the latest EdgeTX v2.10.0 RC+ firmware" the hardware limitations are still in place after all?

**Reply by richardclli** [reply]
Yes, it is supported finally.  Just need some simple hardware mod.

https://github.com/EdgeTX/edgetx/wiki/PL18-%26-PL18EV-Hardware-Mod-for-Complete-EdgeTX-Support

**Reply by DantesInferi** [reply]
"Simple" :D. For what end-user? :D

**Reply by richardclli** [reply]
For many model players, when one can operate a soldering iron properly.

**Reply by richardclli** [reply]
And well, if Flysky will sell modded PL18, that will be great.

**Reply by pfeerick** [reply]
I managed the RTC and DAC/sound mods just fine... and they shouldn't be to hard to most users that can do some soldering... I'm no expert at SMD soldering ;) Admittedly I also did the flash chip upgrade by hand (not hot air), but it is not a *requirement* to have at least minimum functionality. But I can appreciate to other users it can be daunting. If you are part of a club there is someone who you can ask to do it for you... or if it comes to it... could even ask someone at a mobile phone repair shop to do the mods... just show them the instructions... there are possibilities ;)

**Reply by DantesInferi** [reply]
It's a great thing and kudos for keeping PL18 alive. Chip soldering is a bit over my confidence, but there are people that can be found for this. Its more that, it's some people's main radio, so I need to be ready to buy something like Tx16s if I fail miserably anyway.
